### DISCUSSION TITLE
Creating an account is stuck on "Britain"

### DISCUSSION THREAD

**Discussion by arthurzenika** [ovh/manager#6845] [General]
When trying to create an account I couldn't put my french address. The country dropbox was disabled and grey and so I had to search the internet for a British postcode to validate the form (and I did this after the 2nd nag email saying that it was almost done!)

**Comment by JeremyDec** [comment]
Hello @arthurzenika,

Can you bring more information about your issue? 

Also, I'll suggest you to contact our Help Centre (See: https://help.ovhcloud.com/en/, reach "Contact us") in order to describe the problem you're facing.
This way it will be handle by through our support process, which may help up solve your issue.

Regards,
Jeremy
